.github/workflows/build-and-test-on-pr.yml

```diff
@@ -30,7 +26,9 @@ jobs:
         with:
           path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
           key: ${{ runner.os }}-yarn-2-${{ hashFiles('**/package.json') }}
-
+      - name: Get yarn cache directory path
```

Comment:
This new block seems unnecessary, since the cache dir was already used earlier.

Reply:
Remnants of experiments. Fixed.